services/blockchain-indexer/app.js

```diff
@@ -34,8 +34,12 @@ const loggerConf = {
 LoggerConfig(loggerConf);
 
 const logger = Logger();
+const MODULE = {
+	DYNAMIC_REWARD: 'dynamicReward',
+	REWARD: 'reward',
+};
```

Comment:
Please define it in the constants file.

Reply:
It is not being reused. I would prefer to keep it here unless we have plan for it to be reused in other files in near future.